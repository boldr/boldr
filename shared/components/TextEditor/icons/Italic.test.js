import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Italic from './Italic';

describe('<Italic />', () => {
  const wrapper = shallow(<Italic />);
  it('renders <Italic /> without breaking', () => {
    expect(wrapper.find('svg').length).toBe(1);
  });
  it('renders snapshot', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
