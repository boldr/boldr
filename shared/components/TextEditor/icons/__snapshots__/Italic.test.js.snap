exports[`<Italic /> renders snapshot 1`] = `
<svg
  height="18"
  viewBox="0 0 18 18"
  width="18"
  xmlns="http://www.w3.org/2000/svg">
  <path
    d="M7 2v2h2.58l-3.66 8H3v2h8v-2H8.42l3.66-8H15V2z" />
</svg>
`;
